api/http/handler/file/handler.go

```diff
@@ -13,11 +16,21 @@ type Handler struct {
 // NewHandler creates a handler to serve static files.
 func NewHandler(assetPublicPath string) *Handler {
```

Comment:
Can we add a "compression" boolean flag here, so that only uses gzipped when compression is needed?